resources/building-mobile-extensions/building-an-extension.md

```diff
@@ -4,50 +4,56 @@
 
 Before you build an extension, complete the following tasks:
 
-* Ensure that you are using the Experience Platform SDKs.
+* Ensure that you are using the Adobe Experience Platform SDKs.
 
   Extensions extend the behavior of these SDKs.
 
 * Ensure that you can accomplish your goals only by using an extension.
 
   To determine your goals, think about the following questions:
 
-  * Do you need access to data that is not already exposed through the Experience Platform SDKs?    
-  * Do you need to be notified when messages will be sent, or data is being collected by the  Experience Platform SDKs?   
+  * Do you need access to data that is not already exposed via the Adobe Experience Cloud Platform SDKs?    
```

Comment:
Adobe Experience Platform SDKs - same below

Reply:
@emdobrin, we only have to say "Adobe Experience Platform SDKs" once at the beginning of the documentation. After that, all instances should be "Experience Platform SDKs."